dashboards/model/model.libsonnet

```diff
@@ -0,0 +1,295 @@
+{
+  // On record for each k8s kind.
```

Comment:
I presume these written by hand? I don't expect the k8s object model to change much, just worth the mental note that this file will require maintenance when/if new object types are added.

This could also *probably* be extended to non-core CRDs at some point 🤔

Reply:
Yes hand written, and incomplete.